api/mysagw/case/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from mysagw.identity.models import Identity
+from mysagw.models import HistoricalModel, UUIDModel
+
+
+class Case(UUIDModel, HistoricalModel):
```

Comment:
What does this case represent? I find the name a bit confusing, especially as this project also has Caluma around that also has a Case model..

Reply:
Fixed